### ISSUE TITLE
Add self-voicing of WebVTT audio description

### ISSUE THREAD

**Issue by terrill** [ableplayer/ableplayer#345]
Currently, if audio description is provided via a WebVTT file, the description content is exposed in an ARIA live region and announced by screen readers.  This is problematic for at least two reasons: 

1. It's beyond the scope of what screen readers should be doing. If a user needs to read something else while the video is playing, any keystroke interrupts the screen reader while it's announcing the audio description, whereas the video continues to play. Essentially this breaks the audio description and renders the video inaccessible.
2. We have no control over, nor knowledge of, when the screen reader is finished reading the  description. Therefore, if the user has enabled extended description in Preferences (i.e., pausing the video when description starts), we either have to (a) guess when a screen reader might be finished, and resume playback at that point, or (b) require the user to resume playback. 

Both of these problems could be overcome if we were to implement a self-voicing solution.

**Comment by terrill** [comment]
Done as of 4a2d9f7d954d204da9faab4baa9f87795f70f5a0 using the Web Speech API
